src/cleanfigure.m

```diff
@@ -939,6 +939,30 @@ function removeNaNs(meta, handle)
     topRight    = [xLim(2); yLim(2)];
 end
 % =========================================================================
+function data = limitPrecision(data)
+  % Limit the precision of the given data
+  
+  % Only do something if the data is not empty
+  if isempty(data) || all(~isfinite(data))
+      return
```

Comment:
The alternative would be to check if `range` (in your current code) is empty and return early in that case, since that is the actual problem.

In this code, I thought at first that those cases don't have to be treated in a special way, but they do.